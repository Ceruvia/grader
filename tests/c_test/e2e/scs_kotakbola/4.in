378 114 198
B 64
M 39
M 27
B 84
M 68
E