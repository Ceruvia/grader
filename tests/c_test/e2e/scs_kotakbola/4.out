0 134 148
Tidak
